packs/spell-effects/spell-effect-angelic-halo.json

```diff
@@ -8,7 +8,7 @@
         },
         "duration": {
             "expiry": "turn-start",
-            "sustained": true,
```

Comment:
This probably doesn't need a duration at all, since it's granted by the aura.

Reply:
That's tracks, considering we are doing the same for Bless.